samples/GiftedChatReactNativeControl/__tests__/index.spec.ts

```diff
@@ -0,0 +1,7 @@
+// Tests
+
+describe('Sample tests', () => {
```

Comment:
Do we need this test or it is just a sample one which we need to put with our code?

Reply:
It's just a sample test that we need to put in the code.